site.yml

```diff
@@ -79,3 +79,26 @@
   - { role: docker, etcd_client_port1: 2379 }
   - { role: swarm, run_as: master }
   - { role: contiv_network, run_as: master }
+
+# Netplugin master
```

Comment:
just curious, where do we need these host-groups?

I ask because we intentionally limited `netplugin-node` to be a single host-group for bringing up small demo clusters. For bigger clusters we should be using cluster-mgr + `service-master/worker` groups.